src/core_modules/capture-core-utils/rulesEngine/services/VariableService/defaultValues.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import { typeKeys } from '../../constants';
+
+export const defaultValues = {
+    [typeKeys.TEXT]: '',
+    [typeKeys.LONG_TEXT]: '',
+    [typeKeys.PHONE_NUMBER]: '',
+    [typeKeys.EMAIL]: '',
+    [typeKeys.BOOLEAN]: false,
+    [typeKeys.TRUE_ONLY]: false,
```

Comment:
Shouldn't the `TRUE_ONLY` default value should be `true` instead of `false`?

Reply:
"True only"-field really means checkbox, and those are considered unchecked by default, hence `false`.

Reply:
we agreed to keep `false` for now